src/main/scala/services/merging/JdbcMergeServiceClient.scala

```diff
@@ -209,7 +209,7 @@ class JdbcMergeServiceClient(options: JdbcMergeServiceClientOptions,
 
   private def addColumns(targetTableName: String, missingFields: ArcaneSchema): Task[Unit] =
     missingFields match
-      case Nil => ZIO.unit
+      case null => ZIO.unit
```

Comment:
This seems incorrect: we are checking for empty list, not null. You can replace it with `case List()`, not `case null`

Reply:
since ArcaneSchema is a `Seq`, replaced with `case Seq()`